client/src/components/AlertSnackbar/AlertSnackbar.tsx

```diff
@@ -12,14 +13,39 @@ const AlertSnackbar = () => {
   const { open, setOpen, alertMessage, alertSeverity } = useContext(
     AlertContext
   );
+  const [duration, setDuration] = React.useState<number>(0);
   const anchorOrigin: SnackbarOrigin = {
     vertical: "top",
     horizontal: "center",
   };
 
+  const handleClose = (event?: React.SyntheticEvent, reason?: string) => {
```

Comment:
Add some comments around the following lines. 
"open" is not a good parameter name as it is too generic, considering changing it to something more specific